CMakeLists.txt

```diff
@@ -154,37 +156,19 @@ set(dyload_optional Bluray
                     NFS
                     Plist)
 
-# Required by shared objects we link
-set(required_dep_libs EXPAT)
-
 # Required tools
 find_package(TexturePacker REQUIRED)
 find_package(JsonSchemaBuilder REQUIRED)
 
-foreach(dep ${required_dep_libs})
-  find_package(${dep} REQUIRED)
-endforeach()
-
-foreach(dep ${required_deps})
-  core_require_dep(${dep})
-endforeach()
-
-foreach(dep ${optional_deps})
-  core_optional_dep(${dep})
-endforeach()
+core_require_dep("${required_deps}")
```

Comment:
if you want to get rid of the quotes here, you could also define the function without parameter and loop over ARGN